packages/params-middleware/flow-typed/npm/redux-api-middleware_v2.3.x.js

```diff
@@ -0,0 +1,65 @@
+import type { RSAAction } from 'redux-api-middleware';
+
+declare module 'redux-api-middleware' {
```

Comment:
Is there a way to reuse the typings between all of those packages?

Reply:
We can add it later to the flow-typed repository in order to simplify installation but in my opinion each package should have it's own libraries declarations